templates/demo-store/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,22 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
```

Comment:
@rafaelstz can we move this to a new PR? I'm not against adding it, but would love to have a discussion about it in a separate PR.

Reply:
All set @cartogram, I removed it from this PR, and I will create a new PR for the dev container.